python/paddle/fluid/dygraph/static_runner.py

```diff
@@ -0,0 +1,540 @@
+                dtype=each_var.dtype(),
+                name=each_var.name(),
+                type=each_var.type(),
+                persistable=True)
+            if params_filename is None:
+                if not self._is_parameter(each_var):
+                    continue
+                # logging.info("persis var name %s" % each_var.name())
+                framework._dygraph_tracer().trace_op(
+                    type='load',
+                    inputs={},
+                    outputs={'Out': new_var},
+                    attrs={
+                        'file_path': os.path.join(load_dirname, orig_each_name)
+                    })
+                new_var.stop_gradient = False
+                self.add_parameter(name=new_var.name, parameter=new_var)
+                self._param_names.append(new_var.name)
+            else:
+                load_var_map[each_var.name()] = new_var
+
+        if params_filename is not None:
+            load_var_list = []
+            for name in sorted(load_var_map.keys()):
+                load_var_list.append(load_var_map[name])
+
+            framework._dygraph_tracer().trace_op(
+                type='load_combine',
+                inputs={},
+                outputs={'Out': load_var_list},
+                attrs={
+                    'file_path': os.path.join(load_dirname, params_filename)
+                })
+
+            for each_var in persis_vars:
+                if not self._is_parameter(each_var):
+                    continue
+                param = load_var_map[each_var.name()]
+                param.stop_gradient = False
+                self.add_parameter(name=param.name, parameter=param)
+                self._param_names.append(param.name)
+
+    def _recheck_stop_gradients(self):
+        assert self._program_desc is not None, "The StaticModelRunner not initialized properly."
+        # NOTE: After loading the model, the stop_gradient information 
+        # of the original variable is lost, but if a parameter does not
+        # have a corresponding @GRAD variable in the backward program,
+        # it can be said that it is also stop_gradient
+        all_var_names = self._get_all_var_names(self._program_desc)
+        for param_name in self._parameters:
+            param_grad_name = param_name + core.grad_var_suffix()
+            if param_grad_name not in all_var_names:
+                logging.info("set %s stop gradient = True" % param_grad_name)
+                self._parameters[param_name].stop_gradient = True
+
+    def _get_grad_vartype(self, name):
+        assert self._program_desc is not None, "The StaticModelRunner not initialized properly."
+        grad_name = name + core.grad_var_suffix()
+        for i in six.moves.range(self._program_desc.num_blocks()):
+            block = self._program_desc.block(i)
+            var_desc = block.find_var_recursive(cpt.to_bytes(grad_name))
+            return var_desc.type() if var_desc is not None else None
```

Comment:
Continue or return if var_desc is None?

Reply:
`return if var_desc is None` = `return var_desc.type() if var_desc is not None else None`? if 
 use continue, this function need to move to forward

Reply:
done, thanks